src/runtime_src/core/edge/drm/zocl/zocl_drv.c

```diff
@@ -653,20 +653,18 @@ static int zocl_drm_platform_probe(struct platform_device *pdev)
 		zdev->ert = (struct zocl_ert_dev *)platform_get_drvdata(subdev);
 	}
 
+	/* For Non PR platform, there is not need to have FPGA manager
+	 * For PR platform, the FPGA manager is required. No good way to
+	 * determin if it is a PR platform at probe.
+	 */
 	fnode = of_get_child_by_name(of_root,
 	    zdev->zdev_data_info->fpga_driver_name);
 	if (fnode) {
 		zdev->fpga_mgr = of_fpga_mgr_get(fnode);
-		if (IS_ERR(zdev->fpga_mgr)) {
-			DRM_ERROR("FPGA Manager not found %ld\n",
-			    PTR_ERR(zdev->fpga_mgr));
+		if (IS_ERR(zdev->fpga_mgr))
 			zdev->fpga_mgr = NULL;
-		}
 		DRM_INFO("FPGA programming device %s founded.\n",
 		    zdev->zdev_data_info->fpga_driver_name);
-	} else {
-		DRM_ERROR("FPGA programming device %s not found\n",
-		    zdev->zdev_data_info->fpga_driver_name);
 	}
 
 	if (of_property_read_u32(pdev->dev.of_node, "xlnx,pr-isolation-addr",
```

Comment:
Is this the way to determine if it is a PR platform? For non-PR platform, if we still have fpga-mgr configured in device-tree, it seems an error too. What if we move this line above the fpga-mgr code, and print errors for PR platform only?

Reply:
For non-PR platform, the FPGA manager is not used by us. But it could be used for other purpose, such as reprogram the whole PL. In this case, it is not an error. If we have solid way to know this is a PR platform, we could print error if fgpa-mgr is not found.

Reply:
ok